RomeALincei/ANLcr2.xml

```diff
@@ -170,7 +172,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 
                                 <msItem xml:id="ms_i1.3.3">
                                     <locus from="30rb" to="30vb"/>
-                                    <title>Miracle of the Three Youths in the Furnace</title>
+                                    <title ref="">Miracle of the Three Youths in the Furnace</title>
```

Comment:
Has to be created, i think in the record 1295 there only was a commemoration on the youths